### PR TITLE
updated with the troubleshooting step for missing Github secret prior…

### PR DESCRIPTION
**Pull request by aadeshpa**
… to run any kabanero pipeline

### CODE REVIEW

File: Troubleshooting.md [hunk 1]
```diff
@@ -50,4 +51,18 @@
      NOTE: You should see two persistent volumes as shown above, `registry-volume` is by default present for you and `manual-pipeline-run-pvc` 
      is supposed to be applied as manual pre-requisite step before running any pipelinerun. Follow steps [here](https://github.com/kabanero-io/kabanero-pipelines/blob/master/README.md#create-a-persistent-volume)
     
-   
+2. I get the below error while running the building step of any kabanero pipeline if I forget to do below three things as pre-requisite
```
Comment (tseelbach) [inline Troubleshooting.md]: I suggest this text:  I see the error below while running the building step of a Kabanero Pipeline.  What causes this error?

File: Troubleshooting.md [hunk 2]
```diff
@@ -50,4 +51,18 @@
...
+{"level":"fatal","ts":1568057940.179904,"logger":"fallback-logger","caller":"git-init/main.go:37","msg":"Error fetching git repository: exit status 128","stacktrace":"main.main\n\t/go/src/github.com/tektoncd/pipeline/cmd/git-init/main.go:37\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:198"}
+
+Step failed
+
+```
+ - Throubleshooting steps
+  - Keep in mind that When running your Tekton Pipelines, if you see a `fatal: could not read Username for *GitHub  repository*: No such device or address` message in your failing Task logs, this indicates there is no `tekton.dev/git`  annotated GitHub secret in use by the ServiceAccount that launched this PipelineRun. It is advised to create one through the Tekton Dashboard. The annotation will be added and the specified ServiceAccount will be patched.
```
Comment (tseelbach) [inline Troubleshooting.md]: Delete "Keep in Mind that"
Change "It is advised to create one through" to "You need to create one via the Tekton Dashboard"

Is it obvious how to do that in the dashboard?   If not then add more detail.

**Comment by aadeshpa** [comment]
Thanks @tseelbach : made the changes.

File: Troubleshooting.md [hunk 3]
```diff
@@ -50,4 +51,22 @@
...
+{"level":"fatal","ts":1568057940.179904,"logger":"fallback-logger","caller":"git-init/main.go:37","msg":"Error fetching git repository: exit status 128","stacktrace":"main.main\n\t/go/src/github.com/tektoncd/pipeline/cmd/git-init/main.go:37\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:198"}
+
+Step failed
+
+```
+ - Throubleshooting steps
+  - When running your Tekton Pipelines, if you see a `fatal: could not read Username for *GitHub  repository*: No such device or address` message in your failing Task logs, this indicates there is no `tekton.dev/git`  annotated GitHub secret in use by the ServiceAccount that launched this PipelineRun. You need to create one via the Tekton Dashboard. The annotation will be added and the specified ServiceAccount will be patched.
```
Comment (tseelbach) [inline Troubleshooting.md]: When i look at "rich diff" this troubleshooting section is not displayed correctly.  Probable because of line 70 ```

Comment (aadeshpa) [reply Troubleshooting.md]: correct, updated with correct indentation and removal of line 70

**Pull request merged**